dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/OfficialName/RequireAuthenticationMilestoneAttribute.cs

```diff
@@ -0,0 +1,45 @@
+using System.Security.Claims;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using TeacherIdentity.AuthServer.Services.DqtApi;
+
+namespace TeacherIdentity.AuthServer.Pages.Account.OfficialName;
+
+[AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+public class OfficialNameChangeEnabledAttribute : Attribute, IAsyncPageFilter, IOrderedFilter
```

Comment:
(File name seems off here too)
```suggestion
public class CheckOfficialNameChangeIsEnabledAttribute : Attribute, IAsyncPageFilter, IOrderedFilter
```